lib/doorkeeper/openid_connect/config.rb

```diff
@@ -120,6 +120,12 @@ def extended(base)
       option :expiration, default: 120
 
       option :claims, builder_class: ClaimsBuilder
+
+      option :protocol
+
+      def protocol
+        @protocol ||= ::Rails.env.production? ? :https : :http
+      end
```

Comment:
This is the only way that I could figure to provide a default. Using the `default` option means that the value is only set when `lib/openid_connect/config` is required. In practice this should be fine, but it's also hard to test.